src/contacts/model/ContactUtils.ts

```diff
@@ -28,33 +28,12 @@ export function getContactListName(contact: Contact): string {
 
 export function formatBirthdayNumeric(birthday: Birthday): string {
 	if (birthday.year) {
-		//in chromimum Intl.DateTimeFormat is buggy for some dates with years the format subtracts a day from the date
-		//example date is 15.8.1911 ->format returns 14.8.1911
-		//this issue does not happen with recent years so the formatting is done with the current year then this year is changed with the original of the birthday
-		let refYear = new Date()
-		let bdayString = formatDate(new Date(refYear.getFullYear(), Number(neverNull(birthday).month) - 1, Number(neverNull(birthday).day)))
-		bdayString = bdayString.replace(/\d{4}/g, String(neverNull(birthday).year))
-		return bdayString
+		return formatDate(new Date(Number(neverNull(birthday).year), Number(neverNull(birthday).month) - 1, Number(neverNull(birthday).day)))
```

Comment:
I know you didn't add it but this `neverNull` is not needed anywhere in this function